blaise82/l.m.s-client
# #172019621 Admin should be able to add a book category

#### What does this PR do?
- Allows an admin to add a book category or section
#### Description of Task to be completed?
-  Create a book section interface using ```React```
- Manage state using ```Redux``` and ```redux thunk```
- Integrate the created front end with a hosted API on this URL: ```https://lms-server-octopus.herokuapp.com/api/v1/sections``` 
#### How should this be manually tested?
- Clone the repo
- Run ```npm run install``` in terminal to get ready all app dependencies
- Run ```npm test``` in the terminal to see if all unit tests are working
- Run ```npm start``` to fire up the local server
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#172019621](https://www.pivotaltracker.com/story/show/172019621)
### Screenshots (if appropriate)
<img width="1356" alt="Screen Shot 2020-04-01 at 17 30 12" src="https://user-images.githubusercontent.com/51243858/78157692-8ec18e80-7440-11ea-95a2-601c3511763a.png">
<img width="1360" alt="Screen Shot 2020-04-10 at 04 15 29" src="https://user-images.githubusercontent.com/51243858/78956869-f8e2cf00-7ae3-11ea-8e76-e7d706b0974c.png">
<img width="1349" alt="Screen Shot 2020-04-10 at 04 15 16" src="https://user-images.githubusercontent.com/51243858/78956878-01d3a080-7ae4-11ea-86a1-1d4a99d59a51.png">


#### Questions:
N/A
